pragma-sdk/pragma_sdk/onchain/mixins/merkle_feed.py

```diff
@@ -14,5 +17,28 @@ class MerkleFeedMixin:
     account: Account
     summary_stats: Contract
 
-    async def i_do_not_know_yet(self) -> None:
-        raise NotImplementedError("🙅‍♀ i_do_not_know_yet not implemented yet!")
+    async def get_options_data(
+        self,
+        block_id: Optional[BlockId] = "latest",
+    ) -> OptionData:
+        """
+        Returns the latest OptionData.
+        """
+        (response,) = await self.summary_stats.functions["get_options_data"].call(
+            block_number=block_id
+        )
+        return OptionData(**dict(response))
+
+    async def update_options_data(
+        self,
+        merkle_proof: MerkleProof,
+        update_data: OptionData,
+    ) -> None:
+        """
+        Update the Option data upon merkle proof verification.
+        """
+        print(self.summary_stats.functions)
+        (response,) = await self.summary_stats.functions["update_options_data"].call(
```

Comment:
should be invoke, and need to add the execution config param

Reply:
done

Reply:
done